demisto_sdk/commands/lint/tests/test_linter/gather_facts_test.py

```diff
@@ -200,3 +204,28 @@ def ignored(self, files):
     assert files_paths[-1] in runner._facts['lint_files']
     runner._remove_gitignore_files("prompt")
     assert files_paths[-1] not in runner._facts['lint_files']
+
+
+dir_path = tempfile.mkdtemp()
+python_path = f'{dir_path}/__init__.py'
+expected_path = dir_path
+path_list = [python_path, dir_path]
```

Comment:
I prefer you move the init into the test, as you also handling the cleaning of the temp folder.